s3/pkg/service/object.go

```diff
@@ -355,45 +357,77 @@ func (s *s3Service) GetObject(ctx context.Context, req *pb.GetObjectInput, strea
 		return err
 	}
 
-	var finalReader io.Reader
-	finalReader = reader
+	allBytes, _ := ioutil.ReadAll(reader)
+
+	log.Infof("allBytes.len - %v", len(allBytes))
+
+	allDecBytes := make([]byte, 0)
```

Comment:
How to handle with range download when start offset is not aligned?

Reply:
in our GET Object API, we do not have or support start offset based GET. So, not an issue, in my understanding

Reply:
I think user can specify range in header to get a specific range, not need new API.

Reply:
Ok